src/utils/couchSchema.ts

```diff
@@ -1,10 +1,14 @@
-/* eslint-disable no-var */
 import { DatabaseSetup } from 'edge-server-tools'
 
-import { mockPlugins } from './mockPlugins'
+import { config } from './config'
+import { getLowercaseSwapPlugins } from './lowercaseSwapPlugins'
 
 export const minAmtDbPrefix = 'minamount_'
 
-export const couchSchema: DatabaseSetup[] = mockPlugins.map(plugin => {
-  return { name: minAmtDbPrefix + plugin.pluginName }
-})
+const swapPluginNamesArr = getLowercaseSwapPlugins(config.plugins)
```

Comment:
Are you adding the plugins to the config manually?

Reply:
Yes, the plugins that are available for the account to use have to be added to the config manually.